composer.json

```diff
@@ -19,8 +19,8 @@
         "vlucas/phpdotenv": "^4|^5"
     },
     "require-dev": {
-        "composer/composer": "^2.7.7",
```

Comment:
Do we need to disable bumping in renovate?

Reply:
maybe, I need to look into this some more - I wasn't able to install drainpipe-dev following our installation instructions with a fresh copy of Drupal. Just wanted to get the PR up in the meantime for some feedback.

Reply:
this should be resolved now